package.json

```diff
@@ -2,9 +2,14 @@
   "name": "drf-js",
   "version": "0.0.0",
   "description": "A Django Rest Framework js wrapper library",
-  "main": "index.js",
+  "main": "drf.js",
   "scripts": {
-    "test": "echo test"
+    "test": "npm run eslint; npm run jscs; npm run karma-single",
+    "eslint": "eslint src",
+    "jscs": "jscs src",
```

Comment:
How about `.` instead of `src` and use ignore files to exclude `node_modules` and the like? Then any new file will be implicitly included in linting, like conf files